aiidalab_widgets_base/utils/__init__.py

```diff
@@ -114,33 +114,39 @@ class _StatusWidgetMixin(traitlets.HasTraits):
     """
 
     message = traitlets.Unicode(default_value="", allow_none=True)
+    new_line = "\n"
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, clear_after=3, *args, **kwargs):
         self._clear_timer = None
+        self._clear_after = clear_after
         self._message_stack = []
         super().__init__(*args, **kwargs)
 
     def _clear_value(self):
         """Set widget .value to be an empty string."""
         if self._message_stack:
             self._message_stack.pop(0)
-            self.value = "<br>".join(self._message_stack)
+            self.value = self.new_line.join(self._message_stack)
         else:
             self.value = ""
 
-    def show_temporary_message(self, value, clear_after=3):
+    def show_temporary_message(self, value):
```

Comment:
Let's please avoid the incompatible change and use the class variable as default value (as suggested multiple times before):
```suggestion
    def show_temporary_message(self, value, clear_after=None):
        clear_after = clear_after or self._clear_after
```